src/Elastic.Apm.MongoDb/Elastic.Apm.MongoDb.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
Hmm, `net461` was included in the original code, that's why it's here. I think we are good to start with `netstandard2.0`. That'll cover also newer Full Framework versions. `net461` is usually added specifically, because `netstandard2.0` doesn't work well on that version. Let's go with `netstandard2.0` only, and if someone needs `net461`, we can add it later.